### ISSUE TITLE
Confusion about the contrastive learning.

### ISSUE THREAD

**Issue by z1358** [Sha-Lab/FEAT#63]
Great work!
I have some confusion about the code implementation on contrastive loss. 
In the paper, it is suggested that the loss acts on `instances embeddings after adaptation`.
However, in `feat.py`, logits_reg is calculated by aux_center and aux_task, where aux_task seems `embeddings before adaptation`?
Looking forward to your reply!

**Comment by Han-Jia** [comment]
Hi, sorry for the confusion in the paper.

The main idea of the regularization part is the adapted prototypes with the transformer should be contrastive, performing better for classification even on the previous embeddings.
